phpunit/functional/Glpi/Form/Destination/CommonITILField/EntityFieldTest.php

```diff
@@ -234,7 +235,7 @@ private function sendFormAndAssertTicketEntity(
         $this->updateItem(
             $destination::getType(),
             $destination->getId(),
-            ['config' => ['entity' => $config->jsonSerialize()]],
+            ['config' => [(new EntityField())->getKey() => $config->jsonSerialize()]],
```

Comment:
I've made the method static a few weeks ago to make accessing the key easier.
Could you apply it everywhere ?

```suggestion
            ['config' => [EntityField::getKey() => $config->jsonSerialize()]],
```